src/components/editTestCase/qdm/EditTestCase.tsx

```diff
@@ -90,6 +97,26 @@ const EditTestCase = () => {
   dayjs.extend(utc);
   dayjs.utc().format(); // utc format
 
+  const featureFlags = useFeatureFlags();
+  const [hasObservationOrStratification, setHasObservationOrStratification] =
+    useState(false);
+  useEffect(() => {
+    if (featureFlags?.disableRunTestCaseWithObservStrat) {
+      const groups: Group[] = measure?.groups;
+      groups?.forEach((group) => {
+        const measureObservations: MeasureObservation[] =
+          group?.measureObservations;
+        const measureStratifications: Stratification[] = group?.stratifications;
+        if (
+          measureObservations ||
```

Comment:
same thing here - let's check the length of observations array

Reply:
@nmorasb done, thanks!